src/sites/signalen/routes.tsx

```diff
@@ -0,0 +1,36 @@
+import { ErrorPage } from '../../shared/pages/ErrorPage'
+import { siteUrl } from './constants.ts'
+import { Add } from './pages/Add.tsx'
+import { Contact } from './pages/Contact.tsx'
+import { Describe } from './pages/Describe.tsx'
+import { SignalenRootPage } from './pages/SignalenRootPage.tsx'
+import { Summary } from './pages/Summary.tsx'
+import { Thanks } from './pages/Thanks.tsx'
+
+export const routes = {
+  path: siteUrl,
+  element: <SignalenRootPage />,
+  errorElement: <ErrorPage />,
+  children: [
+    {
+      element: <Describe />,
+      path: `${siteUrl}beschrijf`,
```

Comment:
Isn’t this the site’s home page?

```suggestion
      index: true,
```

Reply:
It is, but the current Signalen has an odd url path structure. You're always redirected from the index to `/beschrijf`, which I tried to emulate here

Reply:
Alright, I’ve used that url when adding the Signalen prototype to the list on the homepage. The root URL for the prototype then results in an error message.

Feel free to resolve.

Reply:
Thanks! We're probably going to simplify the url structure in Signalen v2, so I don't think that's really necessary